pkg/processor/test/suite/suite.go

```diff
@@ -87,6 +101,80 @@ func (suite *TestSuite) SetupTest() {
 	suite.TestID = xid.New().String()
 }
 
+// BlastHTTP is a stress test suite
+func (suite *TestSuite) BlastHTTP(request StressRequest) bool {
+
+	// Set deployOptions of example function "outputter"
+	deployOptions := suite.GetDeployOptions(request.FunctionName,
+		suite.GetFunctionPath(request.FunctionPath))
+
+	// Configure deployOptipns properties, number of MaxWorkers like in the default stress request - 32
+	deployOptions.FunctionConfig.Meta.Name = fmt.Sprintf("%s-%s", deployOptions.FunctionConfig.Meta.Name, suite.TestID)
+	deployOptions.FunctionConfig.Spec.Build.NuclioSourceDir = suite.GetNuclioSourceDir()
```

Comment:
Remove this line, it's obsolete (GetNuclioSourceDir was used before we compiled Go functions as plugins). I guess you copied it from some place which is also wrong